src/controllers/logging/ChannelLog.cpp

```diff
@@ -9,7 +9,8 @@ ChannelLog::ChannelLog(QString channelName)
 
 QString ChannelLog::channelName() const
 {
-    return this->channelName_;
+    QString lowercaseChannelName_ = this->channelName_.toLower();
+    return lowercaseChannelName_;
```

Comment:
Since the "lowercaseChannelName_" variable stored the "channelName_" variable as lowercase, I added a "_" at the end of it. Is "lowercaseChannelName" better than "lowerCaseChannelName_"?

Reply:
Local variables should not use an underscore at the end, no

Reply:
Okay, thanks. I'll commit the bot's suggestion then.